### PR TITLE
Fix mismatch between Dintero item totals and WooCommerce item totals

### PR DESCRIPTION
**Pull request by andersem**
When a WooCommerce order isn't created via a redirect, it will be created via a callback.

In the order creation, the correct products are fetched from WooCommerce by the line ids given in the Dintero transaction, and then added to the order with the item amount specified by the product.

If the item amount was changed when creating the order, for example if a discount is given, this was not respected in the order creation.

In this pull request we fix this by overriding the product total with `$args`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,289 +1,15 @@
-# Dintero Checkout
```
Comment (andersem) [inline README.md:1]: The README.md file should be for developers, while the README.txt file is for the wordpress store

File: changelog.txt [hunk 2]
```diff
@@ -1,230 +0,0 @@
-*** WooCommerce Dintero Plugin Changelog ***
```
Comment (andersem) [inline changelog.txt:1]: Changelog is in README.txt

File: README.md [hunk 3]
```diff
@@ -1,289 +1,15 @@
...
-* Cart Item price issue resolved and Express checkout from cart removed billing address fields
-
-2020.06.05 - 2
-
-* Payment cancelled issue resolved, Now it redirects to cart page
-
-2020.06.08
-
-* Refund issue resolved
-
```
Comment (frodeaa) [inline README.md:257]: maybe move this to a separate PR?

File: includes/class-wc-dintero-hp-ajax.php [hunk 4]
```diff
@@ -335,9 +335,9 @@ public function dhp_create_order(){
 					if($id && get_product( $id )){
 						
 						$args = array(
-								'total'   => $product['amount'] / 100
+								'total'   => ($product['amount'] - $product['vat_amount']) / 100
 					      );
-						$order->add_product(get_product( $id ),$product['quantity']);
+						$order->add_product(get_product( $id ),$product['quantity'], $args);
```
[the remark this answers is not in the document]
Comment (frodeaa) [reply includes/class-wc-dintero-hp-ajax.php:340]: @andersem I think I'm missing some context to review the pr. How is discount related to the change?

can we have a short discussion about this PR?

[the remark this answers is not in the document]
Comment (frodeaa) [reply includes/class-wc-dintero-hp-ajax.php:340]: https://woocommerce.github.io/code-reference/classes/WC-Abstract-Order.html#method_add_product

**Review by frodeaa — APPROVED** [review]
